packages/components/src/components/post-banner/post-banner.tsx

```diff
@@ -29,7 +30,7 @@ import { BANNER_TYPES, BannerType } from './banner-types';
 export class PostBanner {
   @Element() host: HTMLPostBannerElement;
 
-  @State() bannerId = crypto.randomUUID();
+  @State() bannerId = `p${nanoid(6)}`;
```

Comment:
Why are you using `6`? Is it still safe to avoid collisions?

Reply:
The ticket said to set it with at least 6 characters and I checked for the safety, it'd take 37k IDs for a 1% chance of collision.